client/src/pages/CookiesPolicy.js

```diff
@@ -0,0 +1,197 @@
+import React from "react";
+import styled from "styled-components";
+
+import { theme } from "../constants/theme";
+const { colors } = theme;
+
+const PolicyContainer = styled.div`
+  margin: 3rem 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const TextContainer = styled.div`
+  max-width: 60rem;
+`;
+
+const ListNoIndent = styled.ol`
+  padding: 0;
+  margin: 1.6rem;
+`;
+
+const TermsLink = styled.a`
+  color: ${colors.primary};
+`;
+
+const Date = styled.div`
+  text-align: right;
+`;
+
+export const CookiesPolicy = () => {
+  return (
+    <PolicyContainer>
+      <TextContainer>
+        <h2 className="text-primary display-6">Cookies Policy</h2>
+        <ListNoIndent>
+          <li>
+            WHAT ARE COOKIES?
+            <ol type="I">
```

Comment:
what's the type attribute for?

Reply:
It changes the default ordered list (1., 2., 3.) to either lowercase letter (type="a"), uppercase roman numbers (type="I") or lowercase roman numbers (type="i"). I used the same list types as @EdwinMata in the document he sent me.

Reply:
Ah didn't know about that! Lgtm then